pkg/ddc/eac/dir_quota.go

```diff
@@ -0,0 +1,73 @@
+/*
+  Copyright 2022 The Fluid Authors.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+package eac
+
+import (
+	"github.com/aliyun/alibaba-cloud-sdk-go/sdk"
+	"github.com/aliyun/alibaba-cloud-sdk-go/sdk/auth/credentials"
+	"github.com/aliyun/alibaba-cloud-sdk-go/services/nas"
+)
+
+func (mountInfo MountInfo) SetDirQuota() (response *nas.SetDirQuotaResponse, err error) {
```

Comment:
Why do we have to use opensdk to setDirQuota?

Reply:
> Why do we have to use opensdk to setDirQuota?

@cheyang Removed, we will include a setDirQuota/describeDirQuota cmd tool in later PRs.